pyrogram/client/types/user_and_chats/chat.py

```diff
@@ -754,3 +754,23 @@ def export_invite_link(self):
         """
 
         return self._client.export_invite_link(self.id)
+
+    def mark_chat_unread(self):
```

Comment:
When we add a new method we usually name it like `app.action_on_type`, when it's a bound method we remove "type" because it's redundant (bad: `chat.mark_chat_unread`, good: `chat.mark_unread`).